iothub_client/inc/iothub_device_client.h

```diff
@@ -349,6 +348,155 @@ extern "C"
     */
     MOCKABLE_FUNCTION(, IOTHUB_CLIENT_RESULT, IoTHubDeviceClient_UploadMultipleBlocksToBlobAsync, IOTHUB_DEVICE_CLIENT_HANDLE, iotHubClientHandle, const char*, destinationFileName, IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_CALLBACK_EX, getDataCallbackEx, void*, context);
 
+     /**
+    * @brief    This API creates a new upload within Azure IoT Hub, getting back a correlation-id and a
+    *           SAS URI for the Blob upload to the Azure Storage associated with the Azure IoT Hub.
+    * @remark   It is part of a set of functions for more granular control over Azure IoT-based blob uploads.
+    *           This function is expected to be used along with:
+    *           `IoTHubDeviceClient_CreateUploadContext`
+    *           `IoTHubDeviceClient_AzureStoragePutBlock`
+    *           `IoTHubDeviceClient_AzureStoragePutBlockList`
+    *           `IoTHubDeviceClient_DestroyUploadContext`
+    *           `IoTHubDeviceClient_InitializeUpload`
+    *           `IoTHubDeviceClient_NotifyUploadCompletion`
+    *           For the standard less-granular uploads to blob please use either
```

Comment:
```suggestion
    *           For the standard less-granular uploads to Azure blob storage please use either
```

Reply:
The original text implies the exclusivity between this new set of functions and the previous ones.

Reply:
How about "For less-granular control of uploads to Azure blob storage please use either" ?

Reply:
Done as "For less-granular control of uploads to Azure blob storage please use either".